pyfr/backends/hip/driver.py

```diff
@@ -119,7 +119,8 @@ class HIPWrappers(LibWrapper):
     }
 
     # Constants
-    MemcpyDefault = 4
+    EVENT_DISABLE_TIMING = 2
```

Comment:
Can you add hipEventDefault in case someone wants to diable this feature?

Reply:
Here we would probably want them to use hipEventCreate rather than calling hipEventCreateWithFlags with an unnecessary flags argument.  However, as there are no current consumers of this the function is unwrapped.